src/Relations/BelongsToThrough.php

```diff
@@ -253,6 +247,7 @@ protected function buildDictionary(Collection $results)
     /**
      * Get the results of the relationship.
      *
+     * @inheritDoc
```

Comment:
@calebdw only `@inheritDoc` doesn't work here in combination with the `first()` method below, since `first()` comes from a trait with a generic